[2025-08-13T13:06:54.600+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_etl_pipeline.extract_data manual__2025-08-13T13:06:52.967365+00:00 [queued]>
[2025-08-13T13:06:54.629+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_etl_pipeline.extract_data manual__2025-08-13T13:06:52.967365+00:00 [queued]>
[2025-08-13T13:06:54.631+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-08-13T13:06:54.732+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_data> on 2025-08-13 13:06:52.967365+00:00
[2025-08-13T13:06:54.776+0000] {standard_task_runner.py:57} INFO - Started process 183 to run task
[2025-08-13T13:06:54.788+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'weather_etl_pipeline', 'extract_data', 'manual__2025-08-13T13:06:52.967365+00:00', '--job-id', '52', '--raw', '--subdir', 'DAGS_FOLDER/weather_etl.py', '--cfg-path', '/tmp/tmpsyu449po']
[2025-08-13T13:06:54.798+0000] {standard_task_runner.py:85} INFO - Job 52: Subtask extract_data
[2025-08-13T13:06:54.883+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:193 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-13T13:06:55.036+0000] {task_command.py:416} INFO - Running <TaskInstance: weather_etl_pipeline.extract_data manual__2025-08-13T13:06:52.967365+00:00 [running]> on host 5981a842327b
[2025-08-13T13:06:55.519+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2025-08-13T13:06:52.967365+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-13T13:06:52.967365+00:00'
[2025-08-13T13:06:55.597+0000] {logging_mixin.py:151} INFO - Extracted CSV saved to /opt/***/data/weather_data_extracted.csv
[2025-08-13T13:06:55.601+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-08-13T13:06:55.635+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=weather_etl_pipeline, task_id=extract_data, execution_date=20250813T130652, start_date=20250813T130654, end_date=20250813T130655
[2025-08-13T13:06:55.723+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-08-13T13:06:55.828+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
